### PR TITLE
Add support for user account verification.

### PR DESCRIPTION
**Pull request by ChamathNS**
### Proposed changes in this pull request

$subject

related: https://github.com/wso2-extensions/identity-governance/pull/395

### Related issues
https://github.com/wso2/product-is/issues/9990

### CODE REVIEW

**Comment by mefarazath** [comment]
The new claim added needs to be tracked as part of the migration. Can we create a git issue or make sure it is tracked?

File: features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml [hunk 1]
```diff
@@ -549,6 +549,14 @@
 				<AttributeID>preferredChannel</AttributeID>
 				<Description>Preferred Notification Channel</Description>
 			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/accountConfirmedTime</ClaimURI>
+				<DisplayName>Confirmed Time</DisplayName>
+				<!-- If user store based identity store is used, a proper mapped attribute
+				in your user store must be configured for this. -->
+				<AttributeID>confirmedTime</AttributeID>
```
Comment (IsuraD) [inline features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml:557]: If this is not supported in the embedded ldap we need to add to add it.

**Comment by ChamathNS** [comment]
> The new claim added needs to be tracked as part of the migration. Can we create a git issue or make sure it is tracked?

Tracked with https://github.com/wso2/product-is/issues/9924.

File: features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml (hunk 1 above)
Comment (ChamathNS) [reply features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml:557]: Addressed with https://github.com/wso2-extensions/identity-userstore-ldap/pull/40.

**Review by IsuraD — APPROVED** [review]

**Pull request merged**